project_model_to_task/models/project.py

```diff
@@ -32,7 +32,7 @@ def _authorised_models(self):
         help="Action called to go to the original window.")
     model_reference = fields.Reference(
         selection='_authorised_models')
-    task_origin = fields.Char(compute='_get_origin')
+    task_origin = fields.Char(compute='_compute_get_origin')
```

Comment:
It should be `_compute_task_origin`

Reply:
thanks for the fast response. fix is pushed